src/tests/dictionaryConcepts/container/EditConcept.test.jsx

```diff
@@ -589,4 +589,40 @@ describe('Test suite for mappings on existing concepts', () => {
     wrapper.find('button#removeAnswer').simulate('click');
     expect(props.removeAnswer).toHaveBeenCalledWith(props.selectedAnswers[0].frontEndUniqueKey);
   });
+  it('should call updateSourceEventListener function when editting a mapping to a new  source that is not `CIEL`', () => {
```

Comment:
I do not seem to understand testing that updateSourceEventListener is called yet when you are explicitly calling it yourself in the test.

Reply:
I have adjusted this description accordingly